integrationTests/mock/contracts/elrond.go

```diff
@@ -58,6 +67,26 @@ func (ec *ElrondContract) userRole(caller string, value string, arguments ...str
 	return [][]byte{userRole}, nil
 }
 
+func (ec *ElrondContract) getTokenIdForErc20Address(caller string, value string, arguments ...string) ([][]byte, error) {
+	log.Debug("getTokenIdForErc20Address", "caller", caller, "value", value, "arguments", fmt.Sprintf("%v", arguments))
+
+	if len(arguments) != 1 {
+		return nil, ErrInsufficientArguments
```

Comment:
maybe there are too many arguments. in this case, the error message is misleading